### PR TITLE
Tgstation.Server version 5.14.0

### PR DESCRIPTION
**Pull request by Cyberboss**
# Automated Pull Request

This pull request was generated by our [deployment pipeline](https://github.com/tgstation/tgstation-server/actions/runs/5951748986) as a result of the release of [tgstation-server-v5.14.0](https://github.com/tgstation/tgstation-server/releases/tag/tgstation-server-v5.14.0). Validation was performed as part of the process.

The user account that created this pull request is available to correct any issues.

**_We would like to be verified as the publisher of this software but we cannot find documentation on how to do so._**

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
  - This PR is generated as a direct result of a new release of `tgstation-server` this should be impossible
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
  - Validation is performed as a prerequisite to deployment.
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
  - Manifest installation and uninstallation is performed as a prerequisite to deployment.
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

### CODE REVIEW

**Comment by mdanish-kh** [comment]
> We would like to be verified as the publisher of this software but we cannot find documentation on how to do so.

@Cyberboss https://github.com/microsoft/winget-pkgs/issues/100 is being worked on; It's pending on the business process for it to be finalized. You would be able to take control of the package once that's complete. Right now, the hope is that if you integrate automated CI/CD submission (as you've done here), other submissions for this package would eventually slow down/stop

File: manifests/t/Tgstation/Server/5.14.0/Tgstation.Server.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  InstallerUrl: https://github.com/tgstation/tgstation-server/releases/download/tgstation-server-v5.14.0/tgstation-server-installer.exe
+  InstallerSha256: 97133E67A948155A04D5A06D5128C555D237F92615878A8D818EA54152E51875
+  InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.HostingBundle.6
+  ProductCode: '{D24887FA-3228-4509-B5F3-4E07E349F278}'
```
Comment (mdanish-kh) [inline manifests/t/Tgstation/Server/5.14.0/Tgstation.Server.installer.yaml:20]:
```suggestion
  ProductCode: '{6D870774-F424-4C54-8E36-2697296139AB}'
```

File: manifests/t/Tgstation/Server/5.14.0/Tgstation.Server.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  UnsupportedOSArchitectures:
+  - arm
+  - arm64
+  AppsAndFeaturesEntries:
+  - DisplayName: tgstation-server
+    Publisher: /tg/station 13
+    DisplayVersion: 5.14.0
```
Comment (mdanish-kh) [inline manifests/t/Tgstation/Server/5.14.0/Tgstation.Server.installer.yaml:27]:
```suggestion
```
DisplayVersion should not be used when it's equal to PackageVersion. This causes unnecessary version mapping for the WinGet client. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147. I see this has been used in previous versions as well and should be removed there as well

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by Cyberboss** [comment]
Was about to take care of the legacy rogue `DisplayVersion`s and `ProductCode`s but I see you already have @mdanish-kh, appreciated.

**Pull request merged**